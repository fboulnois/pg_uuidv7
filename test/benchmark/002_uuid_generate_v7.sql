SELECT uuid_generate_v7();
