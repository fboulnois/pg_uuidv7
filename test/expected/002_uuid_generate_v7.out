-- set up variables for testing
\set uuidv7 '018570bb-4a7d-7000-8000-000000000000'
-- test the uuid_generate_v7 function
SELECT uuid_generate_v7() > :'uuidv7';
 ?column? 
----------
 t
(1 row)

